Resources/Textures/Tiles/copyright.txt

```diff
@@ -1,3 +1,7 @@
 Alright because tiles don't use RSIs (yet) I gotta write this out.
 
-floor_steel.png taken from CEV-Eris commit 28e589f0ff72a009adf17db767e90be39054f0f2
+steel.png taken from CEV-Eris commit 28e589f0ff72a009adf17db767e90be39054f0f2
+
+bar.png, laundry.png, mime.png, clown.png and kitchen.png are under CC-BY-SA-3.0
```

Comment:
Which codebase are these from? It's grouped with the tg ones but doesn't have a commit.